files/faban/src/cloud/benchflow/collectors/faban.go

```diff
@@ -67,9 +67,9 @@ func backupHandler(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		panic(err)
 		} 
-	minioKey := minio.GenerateKey(path)
+	minioKey := minio.GenerateKey("faban")
```

Comment:
This way all the faban data are going to have the same `hash`. It was better before, using `path`, so that all the faban data belonging to the same `trial` get the same `hash`.

I propose to use: `faban/`+`fabanPath`+`/`